lib/commands/context.js

```diff
@@ -317,13 +317,13 @@ extensions.onPageChange = async function (pageChangeNotification) {
       let needsPageLoad = (() => {
         let item = (arr) => {
           return _.filter(arr, (obj) => {
```

Comment:
wouldn't be https://lodash.com/docs/4.17.4#find more appropriate for such purpose?

Reply:
It might do the same thing, yes. Not sure about "more appropriate".

Reply:
Trying it out...